doc/theoretical_description_classification.rst

```diff
@@ -58,7 +58,28 @@ However, although this method generally results in small prediction sets, it ten
 for example at the border between two classes.
 
 
-2. Adaptive Prediction Sets (APS)
+2. Top-K
+--------
+
+Introduced by [3], the specificity of the Top-K method is that it will give the same prediction set size for all observations.
```

Comment:
Introduced in? instead of by? [3] is referred into this .rst?

Reply:
done